pagopa/io-app
# feat: [#175900141] Added profile deletion abort tracking

## Short description
MixPanel should track whenever a user aborts the account deletion process.

## How to test
- Go to Profile -> Information ... -> Delete your account
- Follow the procedure and restart the app
- Upon restarting, click "Cancel deletion" on the dialog
- The event should be tracked on MixPanel
